modules/unpickler/src/test/scala/ch/epfl/scala/debugadapter/internal/stacktrace/Scala3UnpicklerTests.scala

```diff
@@ -435,6 +517,7 @@ abstract class Scala3UnpicklerTests(val scalaVersion: ScalaVersion) extends FunS
          |  val x: A = new A {}
          |  def m(a: x.type)(b: x.B): A = a
          |  def m(t: Int !: Int) = 1
+         |  def m4() : [T] => Foo => Bar = ???
```

Comment:
This one looks more realistic:

```suggestion
         |  def m() : [T] => List[T] => Option[T] = ???
```